EstateAdministrationUI.BusinessLogic/Factories/IModelFactory.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EstateAdministrationUI.BusinessLogic.Factories
+{
+    using System.Threading.Tasks;
+    using EstateManagement.DataTransferObjects.Responses;
+    using Models;
+
+    public interface IModelFactory
+    {
+        EstateModel ConvertFrom(EstateResponse source);
+    }
+
+    public class ModelFactory : IModelFactory
+    {
+        public EstateModel ConvertFrom(EstateResponse estateResponse)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename parameter 'estateResponse' to 'source' to match the interface declaration.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)